### DISCUSSION TITLE
[Proposal] The making of a quick start guide like OpenBTS or srsRAN

### DISCUSSION THREAD

**Discussion by ProgrammerIn-wonderland** [magma/magma#11082] [Ideas]
So I am a beginner, moving from OpenBTS, since OpenBTS seems to be a relatively dead project. I was having a hard time making my first project, so I propose that there should be a quick start guide. (If there already is one and I've overlooked it please comment the link to it down below). 
I think something of this format would be really nice https://docs.srsran.com/en/latest/general/source/1_installation.html

It would be nice if the quick start guide had the commands you need to quickly deploy magma core on a singular machine, and also had the required hardware you would need. 

I'm looking forward to discussing this more in the comments of this thread!
Any and all ideas are appreciated 😊

**Comment by alex-xoco** [comment]
Hello @ProgrammerIn-wonderland, have you look into https://docs.magmacore.org/docs/basics/introduction.html
here you will find the steps to deploy Magma components: AGW, ORc8r and FeG

**Comment by panyogesh** [comment]
@ProgrammerIn-wonderland : Are you still looking for it.

**Comment by ProgrammerIn-wonderland** [comment]
Not anymore, sorry
